app/src/org/commcare/activities/RecoveryActivity.java

```diff
@@ -56,7 +56,7 @@ public void onCreateSessionSafe(Bundle savedInstanceState) {
         super.onCreateSessionSafe(savedInstanceState);
         if (savedInstanceState == null) {
             // launching activity, not just changing orientation
-            CommCareUtil.triggerLogSubmission(this);
+            CommCareUtil.triggerLogSubmission(this, false);

```

Comment:
@ctsims  so this PR currently only using `force_logs` in log submission request when it's getting triggered from HQ. I agree that it's a good idea to submit logs in some of these cases specifically  1. Recovery 2. Manual Log submission by going to the menu. Though I wonder whether we will want to use the same `force_logs` flag for all of these cases.  HQ does further processing on getting a log submission request with force_logs set like looking for a DeviceLogRequest and deleting it. So It might be useful for us to differentiate  triggered log submission from all other log requests that we still need to go through even when log submission is diasabled.

Reply:
Got it, it definitely makes sense, and I missed that the force logs boolean is used both internally and sets a literal flag named `force_logs`, which makes sense.

If the boolean has the resulting behavior of triggering the HQ header as well as anything else, this seems totally consistent.